### ISSUE TITLE
Multilingualization of apicurio-data-models

### ISSUE THREAD

**Issue by Yuutakasan** [Apicurio/apicurio-data-models#239]
Multilingualization of apicurio-data-models

I would like to make apicurio-data-models multilingual, but I think I can help you if you decide on the implementation policy.

https://github.com/Apicurio/apicurio-studio/pull/1391

**Comment by EricWittmann** [comment]
I agree this is a gap and something we should address.  I'm open to suggestions regarding the approach.  Whatever we do on this, it needs to be something that will work in both Java and Javascript (both node.js and browser).
